tools/src/wyvern/stdlib/support/CommandLineUtils.java

```diff
@@ -0,0 +1,30 @@
+package wyvern.stdlib.support;
+
+public class CommandLineUtils {
+    private static String[] arguments; // array of command line arguments.
+    private static DynArrayList argumentsArrayList; // array list of command line arguments.
+    public static final CommandLineUtils utils = new CommandLineUtils();
+
+    private CommandLineUtils() {
+    }
+
+    public CommandLineUtils(String[] arguments) {
+        this.arguments = arguments;
+        this.convertToDynArrayList();
+    }
+
+    public String get(int index) {
+        return (String) argumentsArrayList.get(index);
+    }
+
+    public int getLength() {
+        return arguments.length;
+    }
+
+    private void convertToDynArrayList() {
```

Comment:
probably shouldn't be named starting with "convert" - better to start with "set" since the main thing this function does is side-effect the global variable to hold the arguments.